### PR TITLE
Mobile_version

### PR DESCRIPTION
**Pull request by markotuya0**
completed the web page,Added the about me section and portfolio section

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -50,10 +66,123 @@ h3 {
...
+
+.social {
   display: flex;
   justify-content: space-between;
   margin-right: 170px;
   margin-left: -14px;
 }
+
+.project {
+  display: flex;
```
Comment (koomekevo) [inline styles.css:81]: @markotuya0 Kindly change from `display: flex` to `display: grid` as per project requirement.

File: index.html [hunk 2]
```diff
@@ -14,19 +14,109 @@
...
+        <ul class="social">
+          <li><a href="https://twitter.com/mark__anthonny"><img src="./logos/Vector.png" alt="twitter"></a></li>
+          <li><a href="https://www.linkedin.com/in/mark-otuya-6a09a5232/"><img src="./logos/Linkedin.jpg" alt="linkedin"></a></li>
+          <li><a href="https://mail.google.com/mail/u/0/markotuya0@gmail.com"><img src="./logos/Vector.jpg" alt="Gmail"></a></li>
+          <li><a href="https://github.com/markotuya0"><img src="./logos/Vector-2.jpg" alt="github"></a></li>
+        </ul>
+      </section>
+
+      <section class="previous-work">
+        <section class="project">
```
Comment (koomekevo) [inline index.html:33]: @markotuya0 Kindly add a title named "My Recent Works" which uses a `h1` tag.

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @markotuya0, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: Your README is descriptive.
:heavy_check_mark: Your project has no linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] Make your Pull Request more descriptive by adding a title and short summary.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by markotuya0** [comment]
added an extra H1 tag to my Html file & made some changes to my css

File: index.html (hunk 2 above)
Comment (markotuya0) [reply index.html:34]: resolved

**Pull request merged**